normandy/recipes/tests/test_exports.py

```diff
@@ -174,6 +174,30 @@ def test_publish_and_unpublish_are_noop_if_not_enabled(self, requestsmock):
 
         assert len(requestsmock.request_history) == 0
 
+    def test_recipe_as_remotesettings_record(self, mocked_autograph):
+        """Test that recipes are serialized as expected by our clients."""
+
+        recipe = RecipeFactory(
+            name="Test", approver=UserFactory(), enabler=UserFactory(), signed=True
+        )
+
+        record = exports.recipe_as_record(recipe)
+        assert sorted(record.keys()) == ["id", "recipe", "signature"]
```

Comment:
Instead of checking only that the keys exist, how about checking the keys against some sort of schema, or using something like `Whatever` to analyze the contents?

An example of using `Whatever`: https://github.com/mozilla/normandy/blob/2eaf362c061ecc40840d8a61377d2b9851c0ff94/normandy/recipes/tests/api/v3/test_serializers.py#L210-L222

An example of directly checking against a schema: https://github.com/mozilla/normandy/blob/2eaf362c061ecc40840d8a61377d2b9851c0ff94/contract-tests/test_v3_api.py#L39-L56